model/patch/patch.go

```diff
@@ -207,9 +210,9 @@ func TryMarkStarted(versionId string, startTime time.Time) error {
 	}
 	err := UpdateOne(filter, update)
 	if err == mgo.ErrNotFound {
-		return nil
+		return false, nil
 	}
-	return err
+	return true, err
```

Comment:
Does any caller use this err?

Reply:
I think bool+errr is really confusing, even if anyone does use it.

Reply:
Yes, see model/task_lifecycle.go